### PR TITLE
Update metadata creation for unidentified trackers

### PR DESCRIPTION
**Pull request by smalluban**

### CODE REVIEW

File: extension-manifest-v3/src/utils/trackerdb.js [hunk 1]
```diff
@@ -32,10 +34,48 @@ export function getMetadata(request) {
...
+    matches = [
+      {
+        pattern: { key: exception.id, name: exception.id },
+        category: { key: 'unidentified' },
+      },
+    ];
   }
 
   const { category, pattern, organization } = matches[0];
+  const blockByDefault = isCategoryBlockedByDefault(category.key);
```
Comment (smalluban) [inline extension-manifest-v3/src/utils/trackerdb.js:49]: I started with updating stats, but after all this place generates metadata for identifying tracker, and it was also used to now in `adblocker.js` if should block the request, so I moved it here.

If the `shouldBlock` is not required for DNR platforms, we can add here a platform check (not sure if we don't need it there).

Comment (smalluban) [reply extension-manifest-v3/src/utils/trackerdb.js:49]: After all each request was checked in `shouldBlockRequest` in the previous version.

File: extension-manifest-v3/src/utils/trackerdb.js [hunk 2]
```diff
@@ -32,10 +34,48 @@ export function getMetadata(request) {
   }
 
   if (matches.length === 0) {
-    return null;
+    exception = getException(request.domain);
+    if (!exception) return null;
+
+    matches = [
+      {
+        pattern: { key: exception.id, name: exception.id },
```
Comment (chrmod) [inline extension-manifest-v3/src/utils/trackerdb.js:42]: would rather use `request.domain` instead of `exception.id`

Comment (smalluban) [reply extension-manifest-v3/src/utils/trackerdb.js:42]: Ok

**Review by chrmod — APPROVED** [review]

**Pull request merged**